source/openapi3/split.yaml

```diff
@@ -3973,6 +3996,10 @@ components:
           type: string
           description: The Contact the payment will be requested from (`Contact.data.id`)'
           example: de86472c-c027-4735-a6a7-234366a27fc7
+        precheck_funds:
+          type: boolean
+          description: Optional parameter to enforce prechecking of available funds before approving payment request. Authoriser contact must have valid agreement and bank connection. If the available funds for a contact is out of date, the payment request will have a state of `unverified` while a balance check is run. To help avoid this delay, a balance refresh can be run preemptively, see [Contact balance refresh](/#refresh-contact-bank-connection) 
```

Comment:
Maybe not in here, but should we document the different behaviour a bit more clearly. e.g:

* If the bank balance isn't fresh enough, the Payment Request behaves asynchronously and it is suggested you subscribe to the webhooks or poll the payment request until it's either approved or declined:
  * Requested -> `unverified` -> `approved`
  * Requested -> `unverified` -> `declined`
* If the bank balance is fresh enough, the Payment Request behaves synchronously:
  * Requested -> `approved`
  * Requested -> `declined`